op.py

```diff
@@ -92,6 +92,10 @@ def _run_playbook_command(self, playbook):
     def provision(self):
         self._run_playbook_command('playbooks/aws/provision.yml')
 
+    def addon_istio(self):
+        self.playbook_args += ['--tags','addon_istio']
+        self._run_playbook_command('playbooks/aws/provision.yml')
```

Comment:
I think create a new playbook will make things easier here and less error prone (e.g. if something goes wrong, such as a future addition to the tool, that causes the entire `provision.yml` playbook to run, then it will overwrite someone's cluster).

Obviously keep the tags and addition to `provision.yml` so that when someone specifies it in their vars file it can run as a part of the initial provision. But I think a separate playbook that we can call that will call `host_facts_aws` will work better than having it always run (again trying to avoid unintended things down the road)